src/WebJobs.Extensions.DurableTask/WebJobs.Extensions.DurableTask.csproj

```diff
@@ -76,6 +84,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Azure.Services.AppAuthentication" Version="1.0.3" />
```

Comment:
Do we need this as a part of this PR?

Reply:
@TsuyoshiUshio , do you remember what you needed this dependency for?

Reply:
Seems like nothing broke! Removed.

Reply:
The UI just hasn't updated, but this is in fact gone.